test/api/liskApi.js

```diff
@@ -1,28 +1,24 @@
-if (typeof module !== 'undefined' && module.exports) {
```

Comment:
I realised that later, and I've created #202 to rethink this. Browser tests are currently broken anyway because they don't have the sinon script from a cdn.

Also, now we have babel compiling our code, I think a more sensible approach is to test against `dist` in the browser, than to force browser testers to have modern es8-friendly browsers. That way the module issue we have here doesn't even arise.